modules/osdk-bundle-validate-about.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-bundle-validate.adoc
+
+:_content-type: CONCEPT
+[id="osdk-about-bundle-validate_{context}"]
+= About the bundle validate command
+
+While the Operator SDK `scorecard` command can run tests on your Operator based on a configuration file and test images, the `bundle validate` subcommand can validate local bundle directories and remote bundle images for content and structure.
+
+Bundle images are pulled from a remote registry and built locally before they are validated. Local bundle directories must contain Operator metadata and manifests. The bundle metadata and manifests should have a structure similar to the following bundle layout:
+
+.Example bundle layout
+[source,terminal]
+----
+./bundle
+  ├── manifests
+  │   ├── cache.my.domain_memcacheds.yaml
+  │   └── memcached-operator.clusterserviceversion.yaml
+  └── metadata
+      └── annotations.yaml
+----
+
+.`bundle validate` command syntax
+[source,terminal]
+----
+$ operator-sdk bundle validate <bundle_dir_or_image> <flag>
+----
+
+The command completes with an exit code of 0 if it does not detect validation errors. The command completes with an exit code of 1 if it detects validation errors. The command only fails on errors. Bundles with only warnings and no errors can still pass with an exit code of 0.
+
+.Example output
+[source,terminal]
+----
+some output
```

Comment:
I believe you're just using "some output" as a placeholder for the moment, right? No worries if so :+1:

Reply:
It is a placeholder for now until I have some example output.